DiffusionPreprocessing/DiffPreprocPipeline.sh

```diff
@@ -134,6 +134,10 @@ PARAMETERs are: [ ] = optional; < > = user supplied value
   [--b0maxbval=<b0-max-bval>]
                           Volumes with a bvalue smaller than this value will be
                           considered as b0s. Defaults to ${DEFAULT_B0_MAX_BVAL}
+  [--topup-config-file=<filename>]
+                          file containing the FSL topup configuration.
+                          Defaults to b02b0.cnf in the HCP configuration
+                          directory (as defined by ${HCPPIPEDIR_Config}).
   [--printcom=<print-command>]
```

Comment:
Can we move the [--printcom] usage to the end of the usage statement?

Reply:
I've made this change for all diffusion preprocessing scripts.